onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -243,4 +272,8 @@ def export_onnx_model(model_name, opset_version, use_external_data_format, model
                                                 use_external_data_format)
             optimize_onnx_model_by_ort(onnx_model_path, ort_model_path, use_gpu, overwrite, model_fusion_statistics)
 
-    return onnx_model_path, is_valid_onnx_model, config.vocab_size, tokenizer.max_model_input_sizes[model_name]
+    model_name_in_tokenizer = model_name
```

Comment:
instead of doing the tricky conversion, how about 
```
    max_input_sizes = tokenizer.max_model_input_sizes[model_name] if model_name in tokenizer.max_model_input_sizes else 1024
```